CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 ## [Unreleased]
 
 ### Added
-
+- We added a Boolean Expression to validate if the current database location is shared, preventing an exception when Pulling Changes From Shared Database. [#6959](https://github.com/JabRef/jabref/issues/6959)
```

Comment:
The changlog should be understandable for normal users, so please remove the "boolean expression" part.